src/main/java/productDependency/CreateOrderPage.java

```diff
@@ -0,0 +1,1481 @@
+	     * @throws IOException 
+	     */
+	  public void selectProduct5() throws IOException{
+	    	
+	    	CreateOrderPage sp=new CreateOrderPage(driver);
+	    	JavaScriptExec.sleep();
+	    	String ProductName = BasePage.getCellData(xlsxName, sheetName, 6, 5);
+			WebElement selectProduct = driver
+					.findElement(By.xpath("//a[@class='cell double']//*[text()='" + ProductName + "']"));
+			navigateBottom();
+			actions.moveToElement(selectProduct).click().perform();
+			JavaScriptExec.sleep();
+	    }
+    
+	  @FindBy(how = How.XPATH, using = "//*[@id='change--3-update-form']/div[2]/a[3]/span")
+	  //*[@id="change--3-update-form"]/div[2]/a[3]/span
+	
+	  private WebElement clickPlusdependency;
+
+	/**
+	 * Method to click on Products tab after successful login.
+	 * 
+	 * @throws IOException
+	 */
+	public void clickPlusdependency() throws IOException {
+		
+		CreateOrderPage sp = new CreateOrderPage(driver);
+		log.info("Click on plusDependency");
+		Assert.assertTrue(clickCustomerTab.isDisplayed());
+		clickPlusdependency.click();
+		JavaScriptExec.sleep();
+	}
+	
+		@FindBy(how = How.XPATH, using = "//*[@id='edit_change_header_-3']/td[1]")
+		//*[@id="edit_change_header_-3"]/td[1]
+		private WebElement clickParentOrder;
+
+		/**
+		 * Method to click on Products tab after successful login.
+		 * 
+		 * @throws IOException
+		 */
+		public void clickParentOrder() throws IOException {
+			
+			CreateOrderPage sp = new CreateOrderPage(driver);
+			log.info("Click on ParentOrder");
+			Assert.assertTrue(clickParentOrder.isDisplayed());
+			clickParentOrder.click();
+			JavaScriptExec.sleep();
+		}
+		
+		@FindBy(how = How.XPATH, using = "//*[@id='edit_change_header_-3']/td[1]")
+		//*[@id="edit_change_header_-3"]/td[1]
+		
+		private WebElement clickParentOrder1;
+
+		/**
+		 * Method to click on Products tab after successful login.
+		 * 
+		 * @throws IOException
+		 */
+		public void clickParentOrder1() throws IOException {
+			
+			CreateOrderPage sp = new CreateOrderPage(driver);
+			log.info("Click on ParentOrder");
+			Assert.assertTrue(clickParentOrder1.isDisplayed());
+			clickParentOrder1.click();
+			JavaScriptExec.sleep();
+		}
+		
+		@FindBy(how = How.XPATH, using = "//*[@id='edit_change_header_-3']/td[1]")
+		//*[@id="edit_change_header_-3"]/td[1]
+	
+		
+		private WebElement clickParentOrder2;
+
+		/**
+		 * Method to click on Products tab after successful login.
+		 * 
+		 * @throws IOException
+		 */
+		public void clickParentOrder2() throws IOException {
+			
+			CreateOrderPage sp = new CreateOrderPage(driver);
+			log.info("Click on ParentOrder");
+			Assert.assertTrue(clickParentOrder2.isDisplayed());
+			clickParentOrder2.click();
+			JavaScriptExec.sleep();
+		}
+	
+	
+	
+	
+	// @FindBy(how=How.XPATH, using="//*[@id="dependencies-products-change_-3"]/tbody/tr[3]/td[1]/a/strong
+	private WebElement clickDependentProduct;
+
+		/**
+		 * Method to click on dependent product.
+		 * 
+		 * @throws IOException
```

Comment:
@mayankchhajer Please remove the unused WebElements.

Reply:
Done